### PR TITLE
refactor(householdsList): refactor householdsList

### PR DESCRIPTION
**Pull request by maximilianoforlenza**
refactor householdsList component and create dwellingVisits component

### CODE REVIEW

File: src/util/getLocaleTime.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {defaultTimeOptions} from '../constants';
+
+const getLocaleTime = (input, options) => {
+    if (!input) {
+        return '';
+    }
+    return new Date(input).toLocaleTimeString('es-AR', options || defaultTimeOptions);
```
Comment (navarroaxel) [inline src/util/getLocaleTime.js]:
```js
static formatTime(input, options = defaultTimeOptions) {
        if (!input) {
            return '';
        }
        const date = isDate(input) ? input : new Date(input);
        return date.toLocaleTimeString('es-AR', options);
}
```

File: src/util/getLocaleDate.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const getLocaleDate = input => {
+    if (!input) {
+        return '';
+    }
+    return new Date(input).toLocaleDateString('es-AR');
+};
```
Comment (navarroaxel) [inline src/util/getLocaleDate.js]:
```js
static formatDate(input, options) {
        if (!input) {
            return '';
        }
        const date = isDate(input) ? input : new Date(input);
        return date.toLocaleDateString(DateUtilsService.LOCALE, options);
}
```

File: src/util/index.js [hunk 3]
```diff
@@ -26,6 +28,8 @@ export {canAnswerQuestion};
 export {chapterPropTypes};
 export {cleanChildrenQuestions};
 export {getHouseholdHeadName};
+export {getLocaleDate};
+export {getLocaleTime};
```
Comment (navarroaxel) [inline src/util/index.js]: moven al `react-native-commons`

File: src/constants/index.js [hunk 4]
```diff
@@ -28,7 +28,13 @@ const answers = {
     NO: 2
 };
 
+const defaultTimeOptions = {
```
Comment (navarroaxel) [inline src/constants/index.js]: pasar al `DateUtilsService`

**Comment by maximilianoforlenza** [comment]
listo

**Pull request merged**